scripts/security/certs-create-per-user.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+ROOT_CA_DIR=$(realpath $(dirname "${BASH_SOURCE[0]}"))
+echo $ROOT_CA_DIR
```

Comment:
This looks a bit messy in the output.  Options:

1. Delete this line
2. Add better echo, e.g. `echo "Writing credential files in path $ROOT_CA_DIR"`

Reply:
Done in latest commit, i've remove this unnecessary log